src/main/java/com/karankumar/bookproject/backend/goal/CalculateReadingGoal.java

```diff
@@ -77,7 +77,7 @@ private static boolean bookHasFinishDateInThisYear(Book book) {
     }
 
     public static int howFarAheadOrBehindSchedule(int booksToReadThisYear, int booksReadThisYear) {
-        int shouldHaveRead = booksToReadFromStartOfYear(booksToReadThisYear) *
+        int shouldHaveRead = (int)(booksToReadFromStartOfYear(booksToReadThisYear)) *
```

Comment:
Sorry, this slipped past me. Could you please make this method return a `double`?

Reply:
Changing this to double can disturb other tests but and even other methods like calculateProgress in ReadingGoalView.java. Because the values will change?